WildAidDemo/auth_providers/local-userpass.json

```diff
@@ -4,8 +4,10 @@
     "config": {
         "autoConfirm": true,
         "resetFunctionName": "resetFunc",
+        "resetPasswordSubject": "Reset Password Link",
+        "resetPasswordUrl": "http://localhost:3000/#/reset_password",
```

Comment:
@am-MongoDB won't this be wrong for anything other than a local build?

@ayushjainrksh - Can this be a relative URL, like "resetPasswordUrl": "/#/reset_password", ?

Reply:
@Sheeri I don't think it would be possible to make it relative because Realm expects a complete URL that is to be sent to the user via email. It even says `The URL must include a scheme, such as http or https.` and that means it requires a complete URL. We can use the localhost URL as the default URL and a user can update the config as per their requirements. Along with a documentation update describing the changes, we should be good to go.

Reply:
This works for people testing on their own machines, and I don't think that's there's a more useful default to use. I'll add a note to the README to tell people to overwrite it.